test/collection.js

```diff
@@ -52,15 +52,19 @@ describe('Collection interface', function() {
 
     assert.throws(() => {
       collectionRef.doc(false);
-    }, /Argument "documentPath" is not a valid ResourcePath. Path is not a string./);
+    }, /Argument "documentPath" is not a valid ResourcePath. Path must be a non-empty strin./);
```

Comment:
spelling (strin)

Reply:
Fixed (and now we know that the '.' really does match everything).